packages/vite/src/node/plugins/license.ts

```diff
@@ -0,0 +1,150 @@
+import fs from 'node:fs'
+import path from 'node:path'
+import type { Plugin } from '../plugin'
+import { isInNodeModules, sortObjectKeys } from '../utils'
+import type { PackageCache } from '../packages'
+import { findNearestMainPackageData } from '../packages'
+
+export interface LicenseEntry {
+  /**
+   * Package name
+   */
+  name: string
+  /**
+   * Package version
+   */
+  version: string
+  /**
+   * SPDX license identifier (from package.json "license" field)
+   */
+  identifier?: string
+  /**
+   * License file text
+   */
+  text?: string
```

Comment:
Wouldn't it be better to add the author and repository url?

https://github.com/vitejs/vite/blob/3400a5e258a597499c0f0808c8fca4d92eeabc17/packages/create-vite/LICENSE#L154-L155

Reply:
It would be, but I'm aiming at a bare minimum implementation as a start since some folks might not want to show it. It's also tricky to compute the repository and authors for packages.